base/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM ghcr.io/pulp/pulp-ci-centos:latest
 
-RUN pip3 install pulpcore django-extensions pulp-smash@git+https://github.com/pulp/pulp-smash.git
```

Comment:
Why are you removing pulpcore?

Reply:
Good question. I figured, it should be installed from source, no? And it kept working for me.

I generally try to treat pulpcore the same way as it's plugins as much as possible.

Reply:
I have pulpcore and pulp smash installed in the container build for mostly selfish reasons. My internet is really slow, and pre installing those takes care of most of the pip requirements for the plugins, since most plugins don't declare a lot of extra dependencies. This makes the containers start up a lot faster since I don't have to install all the python deps.

Reply:
I guess I need to find a better way to cache the pip packages in the build step, since this is mostly a workaround